charts.go

```diff
@@ -1,5 +1,150 @@
 package appoptics
 
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type Chart struct {
+	ID           int       `json:"id,omitempty"`
+	Name         string    `json:"name,omitempty"`
+	Type         string    `json:"type,omitempty"`
+	Streams      []Stream  `json:"streams,omitempty"`
+	Min          float64   `json:"min,omitempty"`
+	Max          float64   `json:"max,omitempty"`
+	Label        string    `json:"label,omitempty"`
+	RelatedSpace int       `json:"related_space,omitempty"`
+	Threshold    Threshold `json:"threshold,omitempty"`
+}
+
+type Stream struct {
+	ID        int    `json:"id,omitempty"`
+	Metric    string `json:"metric,omitempty"`
+	Composite string `json:"composite,omitempty"`
+	Type      string `json:"type,omitempty"`
+	Tags      []Tag  `json:"tags,omitempty"`
```

Comment:
Can you also add fields here to set: 

- summary_function
- group_by_function
- group_by

and the like?